src/main/java/com/webonise/controllers/LatLngController.java

```diff
@@ -0,0 +1,48 @@
+package com.webonise.controllers;
+
+import com.webonise.models.LatLng;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.springframework.stereotype.Component;
+
+@Component
+public class LatLngController {
+
+    private ObservableList<LatLng> latLngList;
+
+    public LatLngController() {
+        latLngList = FXCollections.observableArrayList();
+    }
+
+    public ObservableList<LatLng> getLatLngList() {
+        return latLngList;
+    }
+
+    public LatLng getNewLatLng() {
+        return new LatLng();
+    }
+
+    public void addToList(LatLng point) {
+        latLngList.add(point);
+    }
+
+    public void updateMarker(LatLng point) {
+        for (int index = 0; index < latLngList.size(); index++) {
+            LatLng current = latLngList.get(index);
+            if (current.getPointNo() == point.getPointNo()) {
```

Comment:
float and doubles are not equal safe in java
http://stackoverflow.com/questions/1088216/whats-wrong-with-using-to-compare-floats-in-java